pkg/daemon/multus/config.go

```diff
@@ -0,0 +1,115 @@
+package multus
+
+import (
+	_ "embed"
+	"fmt"
+	"os"
+	"strings"
+	"time"
+
+	"github.com/rook/rook/pkg/operator/k8sutil"
+	"gopkg.in/yaml.v2"
+)
+
+var (
+	//go:embed config.yaml
+	ConfigYaml string
+)
+
+var (
+	DefaultValidationNamespace = "rook-ceph"
+
+	// 1 mon, 3 osds, 1 mgr, 1 mds, 1 nfs, 1 rgw, 1 rbdmirror, 1 cephfsmirror,
+	// (1 csi provisioner, 1 csi plugin) x3 for rbd, cephfs, and nfs CSI drivers
+	DefaultValidationDaemonsPerNode = 16
+
+	DefaultValidationNginxImage = "nginxinc/nginx-unprivileged:stable-alpine"
+
+	DefaultValidationResourceTimeout = 3 * time.Minute
+)
+
+func init() {
+	// the default namespace is the current namespace the operator pod is running in if possible
+	ns := os.Getenv(k8sutil.PodNamespaceEnvVar)
+	if ns != "" {
+		DefaultValidationNamespace = ns
```

Comment:
The tool should always be running in the expected ns, right? Is the default really just in case of a unit test?

Reply:
This is because I still run the tool locally for testing because it's more work to run in the operator pod.

Reply:
Also, thi behavior isn't new, just moved: https://github.com/rook/rook/pull/12396/files#r1234515862